standalone/content/css/webapp.css

```diff
@@ -297,28 +297,123 @@ html[dir="rtl"] .standalone-info-bar > .media-control-buttons > .btn-hangup:afte
   width: 138px;
 }
 
-/**
-* Expired views
-*/
-.highlight-issue-box {
-  width: 400px;
-  margin: 0 auto;
+.issue-unsupported-browser .issue-box-content {
+  width: 1114px;
+}
+
+.issue-unsupported-browser .issue-box-content .info-panel {
+  float: left;
+}
+
+html[dir="rtl"] .issue-unsupported-browser .issue-box-content .info-panel {
+  float: right;
+}
+
+.issue-unsupported-browser .issue-box-content .info-panel h1:before {
+  width: 60px;
+  height: 58px;
+  top: 50%;
+  transform: translate(0, -50%);
+  left: 0;
+  background-size: 60px 58px;
+}
+
+html[dir="rtl"] .issue-unsupported-browser .issue-box-content .info-panel h1:before {
+  right: 0;
+}
+
+.issue-unsupported-browser .issue-box-content .info-panel h1 {
+  font-size: 22px;
+  padding: 0 0 0 80px;
+  line-height: 26px;
+  text-align: start;
+}
+
+html[dir="rtl"] .issue-unsupported-browser .issue-box-content .info-panel h1 {
+  padding: 0 80px 0;
+}
+
+.issue-unsupported-browser .issue-box-content .info-panel .hello-logo {
+  margin: 0;
+  width: 153px;
+  height: 25px;
+  background: transparent url(../img/Hello_Logo.svg) no-repeat center center;
+  background-size: contain;
+}
+
+.issue-unsupported-browser .issue-box-content .hello-supported-browser {
+  margin-left: 50px;
+  display: inline-block;
+  width: 634px;
+  height: 443px;
+  background: transparent url(../img/BrowserWindow_Minimal.svg) no-repeat center center;
+}
+
+html[dir="rtl"] .issue-unsupported-browser .issue-box-content .hello-supported-browser {
+  margin-right: 50px;
+  margin-left: 0;
+  background: transparent url(../img/BrowserWindow_Minimal_RTL.svg) no-repeat center center;
+}
+
+.issue-unsupported-browser .promote-firefox {
```

Comment:
The promote firefox view, is a component in itself. So ideally, the css here should be for just .promote-firefox so that its usable elsewhere if we need it.